sgv2-restapi/src/main/java/io/stargate/sgv2/restsvc/models/Sgv2Keyspace.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright The Stargate Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.stargate.sgv2.restsvc.models;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.swagger.annotations.ApiModelProperty;
+import java.util.List;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
```

Comment:
This looks like so much of the boilerplate code.. Why aren't we already moving to some other framework where these would be so easier (spring, micronaut, whatever).. If we are refactoring already, I would expect that there we have:

* some kind of the lombok support
* moved to OpenAPI and Swagger 3.0

Sorry but I don't understand why we opted in having same issues as before and actually not improving anything.. This first module should imo set standards and work on important updates wrt to V1.. It's gonna be really hard to move away from this if we do that later on..

Reply:
Thanks for the items above on specific expectations, those are helpful. Please let us know if there are more.

Reply:
@ivansenic I was thinking this would be more incremental. First pass there would be minimal changes to the resources aside from having a more proper abstraction between resource and service (I should not have put as much logic in the resource as I did, instead they should have called a service class). Then once we get everything working we'd refactor to a newer framework.

So the first round of reviews would be pushing down logic. Second round would be replacing persistence calls for gRPC calls. Next we could rip out OSGI and make the service stand alone. And finally we'd work in a new framework. Those last two steps could be combined if we want. I'm not a fan of rewriting everything at once since it make it harder for review and when you're debugging you don't know which of the moving parts actually broke things.

Reply:
Yea I feel that I would favor more the bottom-up approach here, than the top-down.. I do understand the plan, but I feel this last step you mentioned would be then a refactoring on it's own again. I mean I don't have anything against current plan and don't want to stop further actions.. I just see so much potential for improvement and modernizing everything.. And also creating like a standardized way of doing things, even the simple things like package structures f.e.. Anyway, no need to discuss more..